x-pack/plugins/cases/public/components/all_cases/columns.tsx

```diff
@@ -335,26 +346,31 @@ const IconWrapper = styled.span`
     width: 20px !important;
   }
 `;
-export const ExternalServiceColumn: React.FC<Props> = ({ theCase }) => {
+
+export const ExternalServiceColumn: React.FC<Props> = ({ theCase, connectors }) => {
   const { triggersActionsUi } = useKibana().services;
 
   if (theCase.externalService == null) {
     return renderStringField(i18n.NOT_PUSHED, `case-table-column-external-notPushed`);
   }
 
+  const lastPushedConnector: ActionConnector | undefined = connectors.find(
```

Comment:
I just want to double check my thought process here, so if a user pushes a case to Jira and then sets the connector to none the cases list page will still show the external connector a Jira. I think that's the behavior we want. This should only be used on the list page right? For the case detail page we'll still show a none connector?

Reply:
Correct to all 🙂 . We want to show the icon of the last pushed connector. For example, I can push to Jira and then select the IBM Resilient connector without pushing. If I go back to the All cases page the IBM Resilient icon will be shown instead of Jira's.